luma/lcd/const.py

```diff
@@ -6,3 +6,8 @@
 class pcd8544(object):
     DISPLAYON = 0x0C
     DISPLAYOFF = 0x08
+
+
+class st7735(object):
```

Comment:
maybe we can start using `since` [version nr here] from now on. I can't find the sphinx link for the directive now but im sure its there.